backend/src/v5/models/users.js

```diff
@@ -128,7 +128,7 @@ User.getFavourites = async (user, teamspace) => {
 
 User.getAccessibleTeamspaces = async (username) => {
 	const userDoc = await User.getUserByUsername(username, { roles: 1 });
-	return userDoc.roles.map((role) => role.db);
+	return userDoc.roles.flatMap((role) => (role.db !== 'admin' ? [role.db] : []));
```

Comment:
This is probably my doing, but since we're changing this line 😆 
```suggestion
	return userDoc.roles.flatMap(({db}) => (db !== 'admin' ? [db] : []));
```